### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by iuliutza**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.

### Description of the Change
The feature consists of two main parts. The first one regard the management of containers. It is represented by a pop up from which the user is able to see how many containers exist, delete and stop them. The second part is represented by the deployment of apps. The feature involves creating a personalized dockerfile suitable for every users' needs

### Benefits
The benefits come from the ease of using dockerized apps combined with the use of containers simplified through the interface of the Studio 
### Possible Drawbacks
If used by users that are new to the docker concept, this feature may cause troubles.

### Applicable Issues

Needed pictures and formating for status, list of containers and the deploy pop up
### Format

[ x] ran `npm run electron-format`
[ x] ran `npm run browser-format`

### Author
Signed-off-by: Luta Iulia Andreea <lutaiulia@gmail.com>

### CODE REVIEW

File: source/plugins/devices/wyapp/devices/raspberrypi/index.js [hunk 1]
```diff
@@ -76,6 +76,38 @@ export function setup (options, imports, register)
...
+		{
+			let dockerfile = null;
+			// TODO add here the dockerfile
+			if(project.language === 'nodejs')
+			{
+				dockerfile = 'FROM balenalib/raspberrypi3-debian-node:latest';
+			}
+			else{
+				dockerfile = 'FROM balenalib/raspberrypi3-debian-python:latest';
+			}
```
Comment (alexandruradovici) [inline source/plugins/devices/wyapp/devices/raspberrypi/index.js:95]: This should be delegated to the device plugin, each device knows its images.

File: source/plugins/devices/wyapp/devices/raspberrypi/index.js [hunk 2]
```diff
@@ -76,6 +76,38 @@ export function setup (options, imports, register)
...
+			dockerfile += studio.projects.getEnvironmentSetup(project);
+
+			if(project.language === 'nodejs')
+			{
+				dockerfile += 'CMD node main.js\n';
+			}
+			else
+			{
+				dockerfile += 'CMD python3 main.py';
+			}
```
Comment (alexandruradovici) [inline source/plugins/devices/wyapp/devices/raspberrypi/index.js:108]: This should be delegated to the language plugin, each device knows its commands.

File: source/plugins/studio/workspace/views/AboutDialog.vue [hunk 3]
```diff
@@ -23,6 +23,7 @@
 				<p><strong>Teona Severin</strong> - Web File System</p>
 				<p><strong>Ana Marinescu</strong> - Pin Layout</p>
 				<p><strong>Andrei Deatcu</strong> - Resistor Color Code, Schematics</p>
+				<p><strong>Iulia Andreea Luta</strong> - Docker</p>
```
Comment (alexandruradovici) [inline source/plugins/studio/workspace/views/AboutDialog.vue:26]: Move to auhtors section

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: source/plugins/devices/wyapp/devices/raspberrypi/index.js (hunk 1 above)
Comment (iuliutza) [reply source/plugins/devices/wyapp/devices/raspberrypi/index.js:95]: The code is already in the raspberry pi plugin, and the images depend on the used language. In this case shouldn't I delegate this to the language plugin?

Comment (alexandruradovici) [reply source/plugins/devices/wyapp/devices/raspberrypi/index.js]: My mistake.

**Review by alexandruradovici — APPROVED** [review]

**Review by alexandruradovici — APPROVED** [review]

**Pull request merged**